bowling/src/fileReader.cpp

```diff
@@ -0,0 +1,66 @@
+#include <algorithm>
+#include <fstream>
+
+#include "fileReader.hpp"
+
+namespace fs = std::experimental::filesystem;
+
+FileReader::FileReader(const std::string& directory) : directoryPath_(fs::path(directory)), isValid_(false) {
+    
+    if (!checkDirectory()) {
+        return;
+    }
+
+    std::vector<fs::path> files = makeFileList();
+    readFiles(files);
```

Comment:
As we don't need `files` in any other place, then we might use `readFiles(makeFileList());` What do you think?